ci/azure_pipelines/templates/info.yml

```diff
@@ -14,7 +14,7 @@ steps:
         echo "Please check the changes, change the azure pipelines acordingly and update the sha256"
         exit 1
       fi
-      EXPECTED_HASH_CI_FOLDER='d57e56d71bd68fc423e4468e1da6b22e5ade2aa2d7f13c3124aeca7dc577a1b4'
+      EXPECTED_HASH_CI_FOLDER='f002bf42d41505103defcb691f8f8c62a37028944fa34dc61156e4d529daa73a'
```

Comment:
@Kiko-Aumond we don't need to generate a new one here, right? (As Edi already generated one) This doesn't seem to hurt, but I want to double check.

Reply:
I was having errors in the Pipeline because of the hash.  Also, if the hash changes it is because someone in the folder changed, so it was necessary to change the hash.